src/rest-server/src/controllers/v2/group.js

```diff
@@ -104,6 +104,29 @@ const updateGroupExtension = async (req, res, next) => {
   }
 };
 
+const updateGroupExtensionAttr = async (req, res, next) => {
+  try {
+    const groupname = req.params.groupname;
+    const attrs = req.params[0].split('/');
```

Comment:
params[0]？？？？？？？ Can you provide some schema about this request?

Reply:
I want to provide an api to update single key in a nested extension, like `/extension/acls/virtualClusters` or `/extension/acls/virtualClusters` or further `/extension/attr_a/attr_b/attr_c...`. In express, I could only find the way of match route path by '*' and then reference it by `req.params[0]`. Do you have better suggestion?

Reply:
See https://stackoverflow.com/questions/23876045/get-rest-of-the-path-from-requested-route-in-nodejs